src/components/Router/index.tsx

```diff
@@ -1,18 +1,38 @@
 import React from 'react';
-import { BrowserRouter as Router, Redirect, Route, Switch } from 'react-router-dom';
+import { BrowserRouter as Router, Redirect, Switch } from 'react-router-dom';
+
+import Signup from 'screens/Signup';
+import Home from 'screens/Home';
+import Login from 'screens/Login';
 
 import CustomRoute from './components/CustomRoute';
-import { ROUTES, routesList } from './constants';
+import { ROUTES } from './constants';
 
-function CustomRouter() {
-  const renderRoutes = () =>
-    routesList.map(route => <CustomRoute key={route.path} isPrivate={route.private} {...route} />);
+const routesList = [
+  {
+    component: Signup,
+    path: ROUTES.signUp
+  },
+  {
+    component: Home,
+    path: ROUTES.home,
+    private: true
+  },
+  {
+    component: Login,
+    path: ROUTES.login
+  }
+];
 
+const renderRoutes = () =>
+  routesList.map(route => <CustomRoute key={route.path} isPrivate={route.private} {...route} />);
```

Comment:
avoid this render function, is the same as having `multi-comp` on the file, keep it inside the component